docs/ref/contrib/admin/index.txt

```diff
@@ -2429,6 +2429,12 @@ creating your own ``AdminSite`` instance (see below), and changing the
     this class is created as ``django.contrib.admin.site`` and you can
     register your models and ``ModelAdmin`` instances with it.
 
+    .. deprecated:: 1.9
+
+      The ``name`` argument has been deprecated and will be removed in
```

Comment:
has been -> is
also please use 4 space indent
It looks like this deprecation hasn’t been done yet and I'm not sure I buy it. How will `current_app=self.name` be refactored in `sites.py`?